src/main/resources/azure-common.properties

```diff
@@ -67,5 +67,5 @@ identifier.networkSecurityGroups=Microsoft.Network/networkSecurityGroups
 identifier.publicIPAddresses=Microsoft.Network/publicIPAddresses
 identifier.virtualNetworks=Microsoft.Network/virtualNetworks
 identifier.virtualMachines=Microsoft.Compute/virtualMachines
-identifier.virtualMachinesExtensions=Microsoft.Compute/virtualMachines/extensions
+identifier.virtualMachinesExtensions=Virtual machine extension
```

Comment:
@kevin-ortega Thanks for changing it back, I saw why it's needed from [this comment](https://github.com/oracle/weblogic-azure/pull/327/files#r1764273837) in weblogic-tag-pr.